modules/ROOT/pages/branching.adoc

```diff
@@ -0,0 +1,38 @@
+= ELN Branching Events +
+
+== Fedora Branching ==
+Fedora Branching happens every 6 months, when Fedora Rawhide branches for the next release.  Example:  When Fedora Rawhide is branched to F35 and fedora-release is set to f36.
+
+=== Trigger for ELN Pipeline ===
+
+The eln build trigger needs to be updated to the latest version that is in fedora-release.  Using our original example.  We would change this to f36.
+
+Example pull request: https://github.com/fedora-ci/eln-build-trigger/pull/2
+
+=== ELN fedora release and repo ===
+
+==== Build fedora-release first ====
+
+```
+koji tag-build eln fedora-release-35-0.1
```

Comment:
We also need to make sure that the `fedora-release` package has the correct `%{rhel}` value, if this happens to be a release that's going to become the next RHEL.

Reply:
Although I don't disagree, it's always good to check that.
But I believe that should go in the "RHEL Branching" section.

Reply:
You are correct.  This does go in this section, and I have added it.